cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/compute/GcpInstanceResourceBuilder.java

```diff
@@ -209,12 +212,35 @@
             Operation operation = insert.execute();
             verifyOperation(operation, buildableResource);
             updateDiskSetWithInstanceName(auth, computeResources, instance);
+            assignToInstanceGroup(context, group, instance, buildableResource);
```

Comment:
We're calling this for all instances? I don't see any references to what instances are behind the load balancer here.

Reply:
Yes, I've added a check that it does exist before attempting to add it, should we want to, we could not create the groups. however there's probably a small benefit to applying this regardless of load balancing. mostly just for monitoring sake.

Reply:
We should mention this to the wider CB team. It can probably just be a comment in the team room. But I know when Azure there was actually a case where Bryce was trying to organize all instances into what are called availability sets, and there was a specific reason it couldn't be done. We should make sure there isn't a similar restriction in Azure. @brycederriso do you remember the details of that?

Reply:
I don't know about it. maybe @pdarvasi @gergopapi2 or @keyki ?

Reply:
Dan and I followed up with Bryce. Bryce was doing something similar where he was putting all instances into availability sets by instance group. @doktoric told us we couldn't do that, because there was a 150 instance limit on availability sets, and at least one customer had an instance group with more than 150 instances. Dan looked into similar limitations for GCP instance groups. There the limitation vary depending on your configuration, but we can have at least 1,000 instances in an instance group. So I think we're okay there, and won't run into the limitation problem we had on Azure.